src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -534,7 +534,11 @@ void CascadiaSettings::_AppendWslProfiles(std::vector<TerminalApp::Profile>& pro
         if (wlinestream)
         {
             std::wstring distName;
-            std::getline(wlinestream, distName, L' ');
+            std::getline(wlinestream, distName, L'\r');
+            size_t firstChar = distName.find_first_of(L"( ");

```

Comment:
I'd definitely leave a comment referring back to #1168 about how we need to make sure to stop the name at a parentheses, since "(default)" might be included in the output